submissions/akulahere/friends-app/script.js

```diff
@@ -76,38 +86,52 @@ const compareByName = (a, b) => a.name.first.toLowerCase() <= b.name.first.toLow
 const compareByAge = (a, b) => a.dob.age - b.dob.age;
 
 const sortByName = (users) => {
-  if (nameSortButton.dataset.sortType !== 'A-Z') {
-    nameSortButton.dataset.sortType = 'A-Z';
+  if (nameSortButton.classList.contains(ASCENDING_ORDER)) {
     users.sort((a, b) => compareByName(a, b));
   } else {
-    nameSortButton.dataset.sortType = 'Z-A';
     users.sort((a, b) => compareByName(b, a));
   }
   return users;
 };
 
 const sortByAge = (users) => {
-  if (ageSortButton.dataset.sortType !== '1-99') {
-    ageSortButton.dataset.sortType = '1-99';
+  if (ageSortButton.classList.contains(ASCENDING_ORDER)) {
     users.sort((a, b) => compareByAge(a, b));
   } else {
-    ageSortButton.dataset.sortType = '99-1';
     users.sort((a, b) => compareByAge(b, a));
   }
   return users;
 };
 
+const toggleSortIcon = (buttonClicked) => {
+  if (!(buttonClicked.classList.contains(ASCENDING_ORDER) || buttonClicked.classList.contains(DESCENDING_ORDER))) {
+    buttonClicked.classList.add(ASCENDING_ORDER);
+  } else {
+    buttonClicked.classList.toggle(ASCENDING_ORDER);
+    buttonClicked.classList.toggle(DESCENDING_ORDER);
+  }
+  if (buttonClicked.nextElementSibling) {
+    buttonClicked.nextElementSibling.classList.remove(ASCENDING_ORDER);
```

Comment:
Don't use constructions like children[0], firstElementChild, parentNode, nextSibling, etc. In such way, you rely on the order of DOM elements. So in case when you will have changed design - your code will be broken. Which is bad, obviously. Use querySelector or closest, if in event, instead.